scripts/test-turbo-modules.sh

```diff
@@ -401,6 +402,51 @@ run_default() {
         --slug dummy-lib-react-native \
         "$working_dir/dummy-lib-rn"
   # ReactNativeDummyLib fails with "› Must be a valid npm package name"
+  main \
+    --force-new-directory \
+    --keep-directory-on-exit \
+    --ubrn-config "$config" \
+    --builder-bob-version 0.35.1 \
+    --skip-ios \
+    --skip-android \
+    --slug @my-org/react-native-dummy-lib \
+    "$working_dir/@my-org/react-native-dummy-lib"
```

Comment:
The lines below use different indentation. Not that it matters at all of course. 😅

```suggestion
      --force-new-directory \
      --keep-directory-on-exit \
      --ubrn-config "$config" \
      --builder-bob-version 0.35.1 \
      --skip-ios \
      --skip-android \
      --slug @my-org/react-native-dummy-lib \
      "$working_dir/@my-org/react-native-dummy-lib"
```